app/assets/stylesheets/application.scss

```diff
@@ -3,7 +3,6 @@ $hmrc-assets-path: "/disclose-cross-border-arrangements/upload/assets/lib/hmrc-f
 
 @import "lib/govuk-frontend/govuk/base";
 @import "partials/spinner";
-@import "partials/summary";
```

Comment:
See below: This summary css file has been removed as it was an implementation from over a year and a half ago to make the width of a segment on a table smaller. This has since changed and after a conversation with Jill it has been removed to keep the table in line with prototype and design pattern.